### ISSUE TITLE
Notice to users otherwise eligible to upgrade of blocking plugins

### ISSUE THREAD

**Issue by wpnzach** [wpninjas/ninja-forms#3049]
If a user is not able to upgrade due to a non-compatible plugin (like reCaptcha or Ninja Layout), give notice to the user of the offending plugin and guide on how to upgrade to 3.0 by deactivating it.

**Comment by kjohnson** [comment]
@wpnzach can you update this with a list of "non-compatible" plugins?

Anything other than reCAPTCHA of Ninja Layout?

**Comment by wpnzach** [comment]
I don't think we need a list.  Our filter looks for anything that starts with "ninja forms" correct?

**Comment by kjohnson** [comment]
Per conversation with @wpnzach:

These add-ons are already blocking the upgrade process, per the `ninja-forms*` prefix of the plugin name/slug.

As a future step we can list the "blocking" plugins/add-ons in an admin notice, as well as along with the Advanced setting for manually upgrading.
